reaper-adm-extension/src/reaper_adm/admimporter.cpp

```diff
@@ -102,11 +102,23 @@ namespace {
 
   template <typename ExistingT, typename NewT>
   std::vector<std::shared_ptr<NewT const>> getElementsIfNo(std::shared_ptr<adm::Document const> doc) {
-      if(doc->getElements<ExistingT>().size() != 0) {
-          return std::vector<std::shared_ptr<NewT const>>{};
-      }
-      auto elementRange = doc->getElements<NewT>();
-      return std::vector<std::shared_ptr<NewT const>>(elementRange.begin(), elementRange.end());
+      auto possibleParents = doc->getElements<ExistingT>();
+      auto possibleElements = doc->getElements<NewT>();
+      std::vector<std::shared_ptr<NewT const>> orphanedElements(possibleElements.size());
+
+      auto it = std::copy_if(possibleElements.begin(), possibleElements.end(), orphanedElements.begin(),
+                  [&possibleParents](auto e) {
+                      for(auto p : possibleParents) {
+                          auto children = p->template getReferences<NewT>();
+                          for(auto c : children) {
+                              if(c == e) return false;
+                          }
+                      }
+                      return true;
+                  });
```

Comment:
This is where we're getting orphan audioTrackUids; https://github.com/ebu/ear-production-suite/blob/1bba004f8d899c2c045c24136fcde2dd19dd54f6/reaper-adm-extension/src/reaper_adm/admimporter.cpp#L180

... but then we don't do anything with it. Do you know what the intended behaviour is? Were we originally just going to import them anyway? Not sure if it's worth the hassle to fix or just remove

Reply:
That looks like an oversight / there should be an additional applyRoutes() call.

I think starting from "orphaned" AudioTrackUIDs  was intended for importing traditional channel based bwf files that have been marked up with ADM - I think the idea is that you can do this with just the chna chunk and references to common definitions. 

If the extension encounters a file like that, it would import the common definitions and create AudioTrackUids from the chna data, then those uids would get pulled out here and used as the starting point of the route tracer to do import in the normal way (typically you'd end up with a single directspeakers plugin I think).

Reply:
Fixed the consts. Commented out the uid search - will open a new issue for that